src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int randomMax = 100;
```

Comment:
violates java constant name convention, rename it
```suggestion
    private static final int randomMax = 100;
```